README.md

```diff
@@ -14,16 +14,17 @@ The ultimate goal of this project is to allow Minecraft: Bedrock Edition users t
 
 Special thanks to the DragonProxy project for being a trailblazer in protocol translation and for all the team members who have joined us here!
 
-### Currently supporting Minecraft Bedrock 1.20.80 - 1.21.3 and Minecraft Java 1.21
+## Supported Versions
+Geyser is currently supporting Minecraft Bedrock 1.20.80 - 1.21.3 and Minecraft Java Server 1.21
 
-## Setting Up
-Take a look [here](https://wiki.geysermc.org/geyser/setup/) for how to set up Geyser.
+We do not support older version due to most Minecraft Bedrock clients being automatically updated. Please update to the recent version if your Minecraft version is no longer supported by Geyser.
```

Comment:
Let's just link https://geysermc.org/wiki/geyser/supported-versions/, that'll cover the rest of the needed info :p